src/main/java/com/orzechazo/eshop/services/BasketServiceImpl.java

```diff
@@ -32,6 +43,7 @@ public BasketDto updateBasket(BasketDto basketDto) {
         Basket currentBasket = getBasketByBasketId(basketDto.getBasketId());
         Basket updateBasket = basketMapper.basketDtoToBasket(basketDto);
         updateBasket.setId(currentBasket.getId());
+        updateBasket.setProducts(currentBasket.getProducts());
```

Comment:
I wonder if this method is needed. As it is now, it will not update products (because you are getting it from current basket), you have separate methods for that. And the only things left are basketId and totalPrice, which should be calculated by service, so I guess we can remove this method?

Reply:
Yeah I was thinking the same thing, I'm glad you agree. I was thinking about using it to update products but it takes basketDto as a parameter so it would make no sense to use basketDto to add or remove products.